src/pages/Entries/ScrollableCalendar.tsx

```diff
@@ -161,12 +166,12 @@ export default function ScrollableCalendar({
 
   // autoscroll past top loading message on load
   useEffect(() => {
-    // height of loading msg + margin is 100 + 12 = 112
-    scrollRef.current!.scrollTop = 112 + 1; // + 1 needed to avoid triggering top load
+    // height of loading msg + margin is 85 + 20 = 105
+    scrollRef.current!.scrollTop = 105 + 1; // + 1 needed to avoid triggering top load
     setPrevScrollHeight(scrollRef.current!.scrollHeight);
   }, [isLoading]);
```

Comment:
I think this might be causing #16 , so if you can fix it now that would be good. Not absolutely necessary for this PR though. It might also be React router (see #16 )

Reply:
yeah i was trying to experiment with it without much luck, idk if this happened bc the height of the happiness card changed?

Reply:
this didn't happen before with the old cards right?